pom.xml

```diff
@@ -36,6 +36,7 @@
         <module>hazelcast-spring-tests</module>
         <module>hazelcast-build-utils</module>
         <module>hazelcast-sql</module>
+        <module>extensions</module>
```

Comment:
Because something may've changed in the extensions and then tests in dependent modules may fail. Like change in mongo module may make mapstore tests fail